### PR TITLE
Expand the responsibilities of payment-form.js

### PR DESCRIPTION
**Pull request by tomverran**
It now handles the display of the card number & showing / hiding the card change box as well as the submission of the payment form, this is so the logic can be reused on the upcoming digital pack manage my account page

image

### CODE REVIEW

**Comment by tomverran** [comment]
@joelochlann thanks!

File: identity/app/views/fragments/profile/stripe.scala.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+@import views.html.fragments.selectDates
+@()
+<ul class="details-list details-list--lower u-unstyled">
+    <li class="details-list__item">
+        <div class="fieldset__heading">
+            <h2 class="form__heading">
+                    Card details
```
Comment (joelochlann) [inline identity/app/views/fragments/profile/stripe.scala.html]: indentation

File: identity/app/views/fragments/profile/stripe.scala.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            </h2>
+        </div>
+
+        <div class="fieldset__fields u-cf">
+            <div class="form-field">
+                <span class="card-type-display js-mma-card-type"><span class="js-mma-payment-card-text u-h"></span></span>
+                <span id="qa-card-details" class="card-details">
+                        •••• •••• •••• <span class="js-mma-card-last4"></span>
+                    </span>
```
Comment (joelochlann) [inline identity/app/views/fragments/profile/stripe.scala.html]: indentation

**Comment by joelochlann** [comment]
:+1: Looks OK, though most of what I said in #11197 still applies.

**Pull request merged**